RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromProtoTracksEDProducer.cc

```diff
@@ -24,6 +24,7 @@
 
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include <vector>
+#include <cassert>
```

Comment:
My take is that the assert and `#include <cassert>` should be removed (as in https://github.com/makortel/cmssw/commit/2b246348f1aced0bee687c78a83bb200db7a2e8b). There should be no need to add them back. (Was the file in this branch perhaps copied from 74X?)